MdePkg/Include/AArch64/AArch64.h

```diff
@@ -150,9 +150,20 @@
 #define VECTOR_ENTRY(tbl, off)    \
   .org off
 
+//
+// PE targets (i.e. building with CLANGPDB) do not support the ELF style
+// .previous directive and so we need to manually define the section again.
+//
+#ifdef __ELF__
 #define VECTOR_END(tbl)           \
   .org 0x800;                     \
   .previous
+#else
+#define VECTOR_END(tbl)           \
+  .org 0x800;                     \
+  .section .text.##tbl##,"ax";    \
+  .align 3
+#endif

```

Comment:
I'd rather have one copy of the code, even if it means we need to work around the CLANGPDB limitation here.